Artifacts/windows-desired-state-configuration-az-automation/RegisterDscNodeConfiguration.ps1

```diff
@@ -0,0 +1,37 @@
+[CmdletBinding()]
+param (
+    [string]
+    $Machine = $env:COMPUTERNAME,   
+    [string]
+    $ConfigName,
+    $ConfigMode,
+    [Int32]
+    $ConfigMinutes,
+    [Int32]
+    $RefreshMinutes,
+    [Boolean]
+    $Reboot,
+    $AfterReboot,
+    [Boolean]
+    $AllowOverwrite,
+    [string]
+    $RegistrationUrl,
+    [string]
+    $RegistrationKey
+)
+Start-Transcript "C:\Artifacts\dsc.log"
```

Comment:
It would be best, if these logs are available in DevTestLabs as well. Please refer how other artifacts log, if you do in the same way, these logs will be routed all the way to the Artifacts UI.
So you may just need to add a log at the end of this script, that dumps the contents of this file and log it in the standard way.

Reply:
I have updated this script to output the contents of the file as requested.